src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,35 @@
+import { AppBar, Box, Button, Toolbar, useTheme } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+const NavBar: React.FC = () => {
+  const theme = useTheme();
+  const navigate = useNavigate();
+  return (
+    <AppBar
+      sx={{backgroundColor:'white', display: "flex", alignItems: "flex-end" }}
```

Comment:
Using "sx" to write styles is good for a small application. But, it is not scalable.
MUI docs suggest using "Emotion" lib to handle styling.

Reply:
@atmeshwar-classic  Updated the changes can you please review them